### DISCUSSION TITLE
Remove titles from Web folder

### DISCUSSION THREAD

**Discussion by kilmerc** [openaudible/openaudible#907] [Q&A] [answered]
I noticed that when I delete a book from the library and export the web page, the file for the book remains in the existing web page directory. Is the intended behavior that the export webpage only ADDs books not present and not to REMOVE books no longer in the library? Is the best way to just manually delete the files from the web page directory after I delete them from my library or is there a better way? Thanks!

**Comment by openaudible** [comment, marked as answer]
The web directory is supposed to be a backup.. so yeah, definitely a feature!! (Not really.) 

We are re-writing the html export to allow the web directory to work with the main library so there aren't duplicates. So that will fix that.

**Reply by kilmerc** [reply]
Sounds good, thanks for the reply!
